docs/user-guide/user-roadmap-zowe-cli.md

```diff
@@ -1,161 +1,55 @@
-
-   New to Zowe CLI? This overview topic introduces what is Zowe CLI.
-
-- [**Architecture**](../getting-started/zowe-architecture.md#zowe-architecture)
-
-   Review the Zowe architecture to understand how Zowe CLI works in the Zowe framework. 
-
-- [**Zowe CLI FAQs**](../getting-started/zowe_faq.md#zowe-cli-faq)
-   
-   If you have a question, it's a good idea to try the FAQ, which answers the most commonly asked questions about Zowe CLI.
-   
-
-## Quick start
-
-> Zowe skill level: Beginner
-
-- [**Zowe CLI quick start**](../getting-started/cli-getting-started.md)
-   
-   Get started with Zowe CLI quickly and easily.
-
-- [**Blog: Getting Started with Zowe CLI**](https://medium.com/zowe/getting-started-with-zowe-cli-7a29420c6be7)
-
-   This blog enables you to get started with Zowe CLI quickly. 
-
-## Installing
-
-> Zowe skill level: Beginner
-
-- [**System requirements**](../user-guide/systemrequirements-cli.md)
-
-   Review this topic to ensure that your system meets the requirements for installing Zowe CLI. 
-
-- [**Installing Zowe CLI**](../user-guide/cli-installcli.md)
-
-   Follow the steps to install a new release of Zowe CLI.
-
-## Configuring and updating
-
-> Zowe skill level: Intermediate
-
-- [**Configuring Zowe CLI environment variables**](../user-guide/cli-configuringcli-ev.md)
-
-   Explains how to configure Zowe CLI environment variables, such as changing log levels, setting the home directory location, and daemon mode.
+Become familiar with Zowe CLI and its place in the Zowe universe with the following resources:
 
-- [**Updating Zowe CLI**](../user-guide/cli-updatingcli.md)
+- Check out [**Zowe release notes**](../whats-new/release-notes/release-notes-overview.md) to find out about the latest Zowe CLI updates.
 
-   Learn how to update Zowe CLI to a more recent version using different methods. 
+- Watch **Office Hours** videos that outline major features in [Zowe CLI V2](../getting-started/zowe-office-hours.md) and [Zowe CLI V3](../whats-new/zowe-v3-office-hours.md).
 
-## Using Zowe CLI and plug-ins
+- New to Zowe? Pick up terms and basic concepts with the [**Zowe Glossary**](../appendix/zowe-glossary.md).
 
-> Zowe skill level: Intermediate
+## Zowe CLI quick start
 
-- [**Using Zowe CLI**](../user-guide/cli-using-usingcli.md)
+Ready to start using Zowe CLI *right now*? Speed up the installation process with the [**Zowe CLI quick start**](../getting-started/cli-getting-started.md) to get Zowe CLI up and running in your environment sooner.
 
-   Learn about how to use Zowe CLI, including connecting to the mainframe, managing profiles, integrating with API Mediation Layer, and more.
+## Getting the most from Zowe CLI
 
-- [**Zowe CLI extensions and plug-ins**](../user-guide/cli-extending.md)
+The following blogs on the publishing platform Medium go over uses cases and provide recommendations on best practices, providing tips on how to best use Zowe CLI to meet your needs.
```

Comment:
Should be: "... go over **use** cases ..."

Reply:
@adam-wolfe this is why you're the wind beneath my wings 🥇